packages/block-editor/src/components/use-editor-feature/index.js

```diff
@@ -13,6 +13,13 @@ import { useSelect } from '@wordpress/data';
  */
 import { useBlockEditContext } from '../block-edit';
 
+const deprecatedFlags = {
+	'colors.allowCustom': ( settings ) =>
+		settings.disableCustomColors === undefined
+			? undefined
+			: ! settings.disableCustomColors,
+};
```

Comment:
This is where we say which existing block setting maps to which feature path (backward compatibility for block editor settings)